protocol.md

```diff
@@ -73,6 +73,24 @@ Finally, both `device_creation` *block*s are pushed to the *Trustchain*, the *vi
 
 Assuming the pushed *block*s are correct, the *Tanker server* stores them, and initiates an authenticated session for the *physical device*. The just created *physical device* permanently stores its [Device Encryption Key Pair] and [Device Signature Key Pair] in the [Local Encrypted Storage].
 
+### User Enrollment
+
+Prerequisite: The *user* is registered on the *application*, but not on the *Trustchain*.
+The *application* trusts or verified one of the *user*'s [Verification Method](#verification-methods).
+
+[User enrollment](#user-enrollment) is done from the *application server*.
```

Comment:
You can add something like

It must be done when converting an existing user base to migrate an application to Tanker. The feature must be disabled in production mode.